src/HttpKernel/InstallationKernel.php

```diff
@@ -67,7 +67,6 @@ public function canBootRealSystem()
     {
         return file_exists($this->getRootDir().'/config/parameters.yml')
             && file_exists($this->getRootDir().'/../system/config/localconfig.php')
```

Comment:
I have tested it and it is actually impossible to use the install tool without a local configuration file. By the time `Config::preload()` is called, the container does not yet exist, so you cannot set `licenseAccepted` via `contao.localconfig`. And even if we change the order, the helper system only loads the `parameters.yml.dist` file, so you would have to define `contao.localconfig.licenseAccepted` there. Worth the effort?